context-free.md

```diff
@@ -2,21 +2,55 @@
 
 ##  Who is this programming language for?
 
+This programming language is for people who want to be able to quickly make
+pictures that can easily be described as a context-free grammar. Many
+recursively generated pictures fall into this category, such as different kinds
+of fractals.
+
 
 ## What is easy to do in this language? Why is it easy?
 
+It is easy to make pictures that are generated by a context-free grammar. This
+is easy because it is how the language is designed: each shape is made up of
+other shapes recursively.
+
 
 ## What is hard to do in this language? Why is it hard?
 
+It is hard(er) to make non-recursive pictures in this language. This is because
+you have to write out all the steps by hand, as opposed to letting the grammar
+do the work for you.
```

Comment:
I definitely noticed that, I initially tried to draw a heart or something and realized that I was gonna have a better time doing something recursively.